cluster/ubuntu/reconfDocker.sh

```diff
@@ -46,7 +46,9 @@ function config_etcd {
 }
 
 function restart_docker {
-  
+
+  source ~/kube/config-default.sh
```

Comment:
It should work if you move this line to the start of `reconfDocker.sh`, and remove the same line in `config_etcd`. Less duplicated.